cmd/kube-dns/app/options/options.go

```diff
@@ -95,9 +97,44 @@ func (m kubeMasterURLVar) Type() string {
 	return "string"
 }
 
+type federationsVar struct {
+	nameDomainMap map[string]string
+}
+
+func (fv federationsVar) Set(v string) error {
```

Comment:
Aah - I see it's described below.  Perhaps a reference to that description here will help.